experiments/Manifest.toml

```diff
@@ -5,9 +5,9 @@ manifest_format = "2.0"
 project_hash = "973443a0a609649a9289253b6f5bc3de181d9006"
 
 [[deps.ADTypes]]
```

Comment:
can we also remove the `experiments/Manifest` update? I think we should remove this file entirely now that we have `.buildkite/Manifest.toml`, but that's a separate issue

Reply:
good catch!